ls1intum/Artemis
# Chore - exam participation cover tests

### Checklist


### Motivation and Context


- exam client test improvements

### Description

- Implemented client tests
- Removed a TODO which was from July and is probably not going to get updated?
- Removed an else branch which is not reachable, since in a test run, the run automatically starts. Code-wise this means the hasStarted() always returns true and therefore, the else clause can not be reached. For more details see the code changes (line 130 and 132 are decisive).

### Steps for Testing


1. Log in to Artemis
2. Create an exam and participate in it (using a student run and using a test run)
3. Make sure it works properly (the cover page mainly)

### Test Coverage



exam-participation-cover.component.ts: 100%
